src/main/java/com/splunk/ibm/mq/metrics/Metrics.java

```diff
@@ -427,4 +427,52 @@ public static LongGauge createMqManagerMaxHandles(Meter meter) {
         .setDescription("Max open handles")
         .build();
   }
+
+  public static LongCounter createMqChannelActivatedEvent(Meter meter) {
+    return meter
+        .counterBuilder("mq.channel.activated.event")
+        .setUnit("{events}")
+        .setDescription("Channel activated event")
+        .build();
+  }
+
+  public static LongCounter createMqChannelConvErrorEvent(Meter meter) {
```

Comment:
It's not clear to me what "conv" means. Convention? Conversions? Convolution? Maybe we should spell this out....